app/services/item_service.rb

```diff
@@ -0,0 +1,15 @@
+class ItemService
+  def self.items
+   conn = Faraday.new(url: "http://localhost:3000") 
+    response = conn.get("/api/v1/items")
+
+    JSON.parse(response.body, symbolize_names: true)[:data]
+  end
+
+  def self.item(item_id)
+    conn = Faraday.new(url: "http://localhost:3000") 
+    response = conn.get("/api/v1/items/#{item_id}")
+
+    JSON.parse(response.body, symbolize_names: true)[:data]
+  end
+end
```

Comment:
Do not go into the :data root key in your service. Let your facade be responsible for filtering out the JSON response.